std/jvm/src/main/scala/cats/effect/std/DispatcherPlatform.scala

```diff
@@ -62,6 +62,9 @@ private[std] trait DispatcherPlatform[F[_]] { this: Dispatcher[F] =>
       case t: TimeoutException =>
         cancel()
         throw t
+      case t: InterruptedException =>
+        cancel()
+        throw t
```

Comment:
Does this syntax work?

```suggestion
      case t: TimeoutException | InterruptedException =>
        cancel()
        throw t
```

Reply:
Unfortunately, no.

Reply:
It's a bit more sophisticated 😉 thanks for trying.

```suggestion
      case t @ (_: TimeoutException | _: InterruptedException) =>
        cancel()
        throw t
```